tools/bf_autocorres.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python2
+
```

Comment:
I think he means that the file is not meant to be executed. The reason I use it here is so that it is semantically obvious that the file is a `python2` file from the first line.